### DISCUSSION TITLE
Is there any way the do the versioning of site!

### DISCUSSION THREAD

**Discussion by shravanngoswamii** [quarto-dev/quarto-cli#9912] [Q&A] [answered]
What I want to do is save each version of my site when new version of Turing.jl releases! See we have a dropdown to change version in navbar in our old site - https://turinglang.org/v0.31/

If quarto does not provide any feature like this then is it possible to add the dynamic dropdown in navbar for versions, I mean I can implement versioning feature using github actions but then the new version number in navbar should update automatically. As far as I know _quarto.yml does not accept any variables

**Comment by mcanouil** [comment]
There is no built-in support for that so you'll have to come up with your own workflow such as using profiles, publishing multiple websites as subdirectories, generating the dropdown with pre-render scripts, etc.

That's basically the same thing as multilingual websites:
- https://github.com/quarto-dev/quarto-cli/issues/275

**Reply by mcanouil** [reply, marked as answer]
There is even an issue to track this:

- https://github.com/quarto-dev/quarto-cli/issues/474

**Reply by shravanngoswamii** [reply]
Ok, thanks!

**Reply by shravanngoswamii** [reply]
> generating the dropdown with pre-render scripts, etc.

I have never tries these pre-render scripts, where can I learn more about it? I can't find enough docs about it!

**Reply by mcanouil** [reply]
As everything, in the documentation.

https://quarto.org/docs/projects/scripts.html#pre-and-post-render
